README.md

```diff
@@ -10,6 +10,7 @@ We welcome contributions to this repository. Please open a pull request or an is
 |--------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------|----------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| ScRAT  | 📄(Bioinformatics)[Mao et al. 2024](https://academic.oup.com/bioinformatics/article/40/2/btae067/7613064)) | [🛠️GitHub](https://github.com/yuzhenmao/ScRAT) | scRNA-seq       | COMBAT: 836k / immune, human ([CELLxGENE](https://cellxgene.cziscience.com/collections/8f126edf-5405-4731-8374-b5ce11f53e82), Haniffa: 528k / immune, human ([CELLxGENE](https://cellxgene.cziscience.com/collections/ddfad306-714d-4cc0-9985-d9072820c530)), SC4: 1.3M / immune, human ([website](http://covid19.cancer-pku.cn/#/summary)). Only evaluated within each dataset | Cells as tokens                                                                | Encoder                                                                         | None                                                    | Phenotype prediction: aggregated per sample cell embeddings are used to predict sample label (e.g., health condition)                                                                  | None                              | 
```

Comment:
as far as I can see, they don't do SSL pre-training, do they? I think the datasets you listed were only used for supervised training, so following the convention it's enough to state "None". 
Aside from that, looks good to me!

Reply:
They don't, I'll change it. In Supplementary, there is a section on the fine-tuning for transferring predictions from one dataset to another. But this is all on the supervised tasks.